src/bootlint.js

```diff
@@ -964,6 +966,86 @@ var LocationIndex = _location.LocationIndex;
             }
         });
     });
+    addLinter("W015", function lintNewBootstrap($, reporter) {
+        var BOOTSTRAP_V4_ERROR = "Bootstrap version 4 detected. Bootlint currently supports Bootstrap version " + CURRENT_BOOTSTRAP_VERSION;
```

Comment:
To avoid having to update the testcases whenever there's a new patch release of Bootstrap, I think we only state the major version in the second part, e.g. "Bootlint currently only supports Bootstrap v3."